### ISSUE TITLE
add wwctl <node|profile> <add|set> --system-overlay and --runtime-overlay

### ISSUE THREAD

**Issue by anderbubble** [warewulf/warewulf#1495]
### Summary

Add new flags `--system-overlay` and `--runtime-overlay` to `wwctl <node|profile> <add|set>`. Retain current flags (`--runtime` and `--wwinit`) as aliases.

### General information

- [X] I have searched the issues of this repo and believe this is not a duplicate

**Comment by mslacken** [comment]
Sounds good. So you want the old flags just
* keep working
* mark as deprecated, so they will print a deprecation warning (possible with cobra)
* hidden, so that they not show up in help

**Comment by anderbubble** [comment]
> So you want the old flags just [...]

Yep, I agree with those three points. (I hadn't thought about a deprecation warning; but, sounds great, especially if cobra can do it easily.)
